asf_core_data/pipeline/data_joining/merge_install_dates.py

```diff
@@ -157,34 +114,38 @@ def manage_hp_install_dates(df, identifier="UPRN", verbose=False):
 
     df["FIRST_HP_MENTION"] = df[identifier].map(dict(first_hp_mention))
 
-    df["ANY_HP"] = df[identifier].map(
-        dict(df.groupby(identifier)["HP_INSTALLED"].max())
-    )
+    print(df[df["FIRST_HP_MENTION"].notna() & df["HP_INSTALLED"]].shape)
 
-    df["HP_AT_FIRST"] = df[identifier].map(
-        df.loc[df.groupby(identifier)["INSPECTION_DATE"].idxmin()]
-        .set_index(identifier)
-        .to_dict()["HP_INSTALLED"]
-    )
+    # Additional features about heat pump history of property
+    if add_hp_features:
 
-    df["HP_AT_LAST"] = df[identifier].map(
-        df.loc[df.groupby(identifier)["INSPECTION_DATE"].idxmax()]
-        .set_index(identifier)
-        .to_dict()["HP_INSTALLED"]
-    )
+        df["ANY_HP"] = df[identifier].map(
```

Comment:
when I see "ANY_HP" I think of "any kind of heat pump", not "a heat pump has previously been installed".

Reply:
Good point, changed it to `HP_AT_ANY_POINT`